File: ueueo-framework/ueueo-background-jobs-abstractions/src/main/java/com/ueueo/backgroundjobs/BackgroundJobExecutionException.cs
using System;
using System.Runtime.Serialization;

namespace Volo.Abp.BackgroundJobs;

[Serializable]
public class BackgroundJobExecutionException : AbpException
{
    public String JobType;// { get; set; }

    public Object JobArgs;// { get; set; }

    public BackgroundJobExecutionException()
    {

    }

    /**
     * Creates a new <see cref="BackgroundJobExecutionException"/> object.
    */
    public BackgroundJobExecutionException(SerializationInfo serializationInfo, StreamingContext context)
        : base(serializationInfo, context)
    {

    }

    /**
     * Creates a new <see cref="BackgroundJobExecutionException"/> object.
    *
     * <param name="message">Exception message</param>
     * <param name="innerException">Inner exception</param>
     */
    public BackgroundJobExecutionException(String message, Exception innerException)
        : base(message, innerException)
    {

    }
}

File: ueueo-framework/ueueo-background-jobs-abstractions/src/main/java/com/ueueo/backgroundjobs/BackgroundJobExecutionException.java
package com.ueueo.backgroundjobs;

import com.ueueo.exception.BaseException;
import lombok.Data;

@Data
public class BackgroundJobExecutionException extends BaseException {

    private String jobType;

    private Object jobArgs;

    public BackgroundJobExecutionException() {

    }

    /**
     * Creates a new <see cref="BackgroundJobExecutionException"/> object.
     *
     * <param name="message">Exception message</param>
     * <param name="innerException">Inner exception</param>
     */
    public BackgroundJobExecutionException(String message, Exception innerException) {
        super(message, innerException);
    }
}
